src/main/java/hachi/Event.java

```diff
@@ -0,0 +1,59 @@
+package hachi;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import exceptions.DateWrongOrderException;
+
+/**
+ * Represents a Event object, extending the Task object.
+ * Includes two additional fields to track the start and end date.
+ * Overrides its representations in storage and in string format.
+ */
+public class Event extends Task {
+    private LocalDate startDate;
+    private LocalDate endDate;
+
+    /**
+     * Constructor for the Event class.
```

Comment:
I think there should be an empty line between the description and the parameters.